frodo.js

```diff
@@ -1,32 +1,29 @@
 /**
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.